app-front/src/components/common/SelectTerritory.tsx

```
@@ -21,7 +22,7 @@ export default function SelectTerritory(props: { defaultValue: number, onChange:
       
     > 
       { data &&
-        data.map((d, i) => <option key={i} value={d.id}>{d.name}</option>)
+        data.data.map((d, i) => <option key={i} value={d.id}>{d.name}</option>)
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)